src/m365/aad/commands/app/app-get.ts

```diff
@@ -90,11 +89,74 @@ class AadAppGetCommand extends GraphCommand {
       });
   }
 
+  private getAppInfo(appObjectId: string): Promise<AppInfo> {
```

Comment:
The returned value of this method is more than what's in `AppInfo`. Let's use instead the `Application` type from the Graph typings so that we don't need to maintain it ourselves.

Reply:
Ah, forgot about the ms graph types :)